raster/src/main/resources/reference.conf

```diff
@@ -1,11 +0,0 @@
-include "akka-actor-reference"
-include "akka-remote-reference"
-
-# Set the geotrellis catalog here.
-# Required for reading any data from GeoTrellis by name.
-#geotrellis.catalog = "src/test/resources/catalog.json"
-
-akka {
-  log-dead-letters = 0
-  log-dead-letters-during-shutdown = off
-}
```

Comment:
This `reference.conf` should actually be moved to the `engine` project's not-yet-existing `reference.conf`